### DISCUSSION TITLE
Following, Follower and Feed Limit

### DISCUSSION THREAD

**Discussion by aps08** [community/community#13355] [Other Feature Feedback, Questions, & Ideas] [answered]
New Feed section should contain:
1. Content from Followers and following ( updates, new repo etc.)
2. There should be limit to number of post let's say 20, once that feed has reach user should click "LOAD MORE" to see more content.
   otherwise.

**Comment by jfullstackdev** [comment, marked as answer]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

this somehow can limit what you see
